jsapp/kobo/directives/KobocatFormPublisher.Directive.js

```diff
@@ -15,7 +15,7 @@ kobo.directive ('kobocatFormPublisher', ['$api', '$miscUtils', '$routeTo', funct
                     $('button.save-button').removeClass('save-button--deploying');
                     scope.close();
                     $miscUtils.alert('Survey Publishing succeeded');
-                    $routeTo.external(results.published_form_url);
+                    // $routeTo.external(results.published_form_url);

```

Comment:
OK, so I guess we may want to parameterize whether this happens and where it goes in the future, seems good for now, can you add a comment in the code above this line explaining the reasoning you just gave here